x/slashing/keeper/keeper_test.go

```diff
@@ -250,7 +250,6 @@ func TestValidatorDippingInAndOut(t *testing.T) {
 	// another block happened
 	ctx = ctx.WithBlockHeight(height)
 	app.SlashingKeeper.HandleValidatorSignature(ctx, val.Address(), newPower, true)
-	height++
```

Comment:
Why remove this?
Is there any problem if I remove this line?

Reply:
Then, I think it's better to remove now. Let's add again when we need